src-tauri/src/northstar/mod.rs

```diff
@@ -235,19 +235,25 @@ pub fn launch_northstar_steam(game_install: GameInstall) -> Result<String, Strin
     }
 
     match steamlocate::SteamDir::locate() {
-        Some(mut steamdir) => {
+        Ok(steamdir) => {
             if get_host_os() != "windows" {
-                match steamdir.compat_tool(&thermite::TITANFALL2_STEAM_ID) {
-                    Some(_) => {}
-                    None => {
-                        return Err(
-                            "Titanfall2 was not configured to use a compatibility tool".to_string()
-                        );
+                match steamdir.compat_tool_mapping() {
+                    Ok(map) => match map.get(&thermite::TITANFALL2_STEAM_ID) {
+                        Some(_) => {}
+                        _ => {
```

Comment:
I'm curious, is there a reason to do `_` over `None`? ^^

Reply:
no good reason outside of rust induced brainrot
fixed by b45d49c8046657c51110d512cb0dbc9ebc53c202